### ISSUE TITLE
ERROR displayed when no primary findings in a variant interpretations

### ISSUE THREAD

**Issue by pamag** [opencb/jsorolla#230]
## Description
bug in variant interpretation when there are no variants in the primary finding
![image](https://user-images.githubusercontent.com/167875/140245853-7f9378d8-9912-41c3-83bb-325509f83666.png)

## Proposed change

make a conspicuous warning that should be only the warning symbol and when clicked it says:
```
Warning:
There are not variants or annotations for this clinical interpretation.
```

**Comment by pamagz** [comment]
see ticket #236

**Comment by jmjuanes** [comment]
Proposed UI change: hide the `variant-interpreter-grid` (and the buttons) and the `variant-interpreter-detail` components and display a warning message as follows: 

![image](https://user-images.githubusercontent.com/5751201/147925481-f63ff05a-8f4f-47f7-8278-7bb79dcf2c9b.png)

**Comment by jmjuanes** [comment]
Warning message implemented in commit 02af20b: 

![image](https://user-images.githubusercontent.com/5751201/148541397-7c20b802-bd1c-4fea-834d-6aadbc21040b.png)

**Comment by pamag** [comment]
## Reviewed

FIXED
